algorithmic_efficiency/workloads/wmt/workload.py

```diff
@@ -116,6 +116,11 @@ def _build_input_queue(self,
     # when _build_input_queue is called (not when next() is first called on it).
     def _input_queue_generator():
       for batch in iter(ds):
+        weights = batch.get('weights')
```

Comment:
I think using `.get()` is correct here, since we do not expect the training batches to already have the key `weights` but validation/test set batches might have it (the remainder batches are padded).

Reply:
yep you're right